src/ui/pages/GameUI.jsx

```diff
@@ -52,7 +63,16 @@ const GameUI = () => {
     <div css={wrapper}>
       <PageHeader>Level 1</PageHeader>{' '}
       <div css={gameBorder}>
-        <canvas css={{ width: '1024px', height: '608px' }} height={608} width={1024} id='GameCanvas' ref={ref} />
+        <canvas css={{ width: '1024px', height: '512' }} height={512} width={1024} id='GameCanvas' ref={ref} />
```

Comment:
```suggestion
        <canvas css={{ width: '50vw' }} height={512} width={1024} id='GameCanvas' ref={ref} />
```

Patrz: sugestia zmiany wyżej + mój wcześniejszy komentarz. Możesz tu w CSS ustawić `width` (i wedle potrzeby `min-`/`max-width`), jednocześnie nie zmieniając atrybutów HTML `width` i `height`. Canvas powinien zachować aspect-ratio a mieć zmienną szerokość.

(+ poprzednio był błąd: `'512'` nie ma jednostek)